deploy.sh

```diff
@@ -221,6 +233,25 @@ fi
 
 log_endgroup
 
+DOJO_HOST_CONFIG=$(docker exec "$DOJO_CONTAINER" sh -c 'echo -n "${DOJO_HOST-}"')
+WORKSPACE_HOST_CONFIG=$(docker exec "$DOJO_CONTAINER" sh -c 'echo -n "${WORKSPACE_HOST-}"')
+
+if [ -z "$DOJO_HOST_CONFIG" ]; then
+        DOJO_HOST_CONFIG="localhost.pwn.college"
+fi
+
+if [ -z "$WORKSPACE_HOST_CONFIG" ]; then
+        WORKSPACE_HOST_CONFIG="workspace.${DOJO_HOST_CONFIG}"
+fi
+
+if [[ "$DOJO_HOST_CONFIG" != "$CONTAINER_IP" && "$DOJO_HOST_CONFIG" =~ [A-Za-z] ]]; then
+        TEST_CONTAINER_EXTRA_ARGS+=("--add-host" "${DOJO_HOST_CONFIG}:${CONTAINER_IP}")
```

Comment:
This helps make DNS work for testing, so tests can resolve IPs. This is for the main node.